Sources/StreamChatSwiftUI/ChatChannel/MessageList/ReactionsIconProvider.swift

```
@@ -19,7 +18,9 @@ class ReactionsIconProvider {
 
     static func color(for reaction: MessageReactionType, userReactionIDs: Set<MessageReactionType>) -> Color? {
         let containsUserReaction = userReactionIDs.contains(reaction)
-        let color = containsUserReaction ? colors.reactionCurrentUserColor : colors.reactionOtherUserColor
+        let color = containsUserReaction ?
+            InjectedValues[\.colors].reactionCurrentUserColor :
+            InjectedValues[\.colors].reactionOtherUserColor
```

Color is struct and reactionCurrentUserColor is lazy var which triggers mutation. Swift 6 was not happy if colors was static var since then static var needs to be made concurrency safe. Ignored the error by switching to accessing injected values directly.